engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -890,4 +880,14 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
+    implicit val actorSystem = CromwellServer.actorSystem
+    val materializedWfDescriptorFuture =
+      PromiseActor.askNoTimeout(MaterializeWorkflowDescriptorActor.MaterializeWorkflow(id, workflowSourceFiles), CromwellServer.workflowDescriptorMaterializeActor).mapTo[MaterializationResult]
+    materializedWfDescriptorFuture map {
+      case MaterializationSuccess(workflowDescriptor) => workflowDescriptor
+      case MaterializationFailure(error) => throw error
+    }
+  }
```

Comment:
Is this implicit class necessary? I'm not a huge fan of implicit classes unless it absolutely makes things better and cleaner. I don't see many advantages of this here currently :confused: